docs/powerauth-server/PowerAuth-Server-1.3.0.md

```diff
@@ -106,6 +106,147 @@ CREATE UNIQUE INDEX pa_application_name_index
     ON pa_application_version (application_id, name);
 ```
 
+## Create Auditing Tables
+
+Create tables for auditing:
+- `audit_log` - table used to store audit logs
+- `audit_param` - table used to store detailed parameters for audit logs
+
+### Oracle
+
+```sql
+--
+-- Create audit log table.
+--
+CREATE TABLE audit_log (
```

Comment:
`arch_db_structure.png` will be updated in a separate PR?

Reply:
Yes, the tables are from our separate auditing projects... We will need to explain this to the users, having "universal anything" is issue, because you either do not document it anywhere, or you copy paste to every repository.